$OpenBSD: patch-CMakeLists_txt,v 1.10 2019/09/22 13:13:48 sthen Exp $

Avoid the broken libwsutil.so that gets generated with -pie -shared
(probably following tools/clang/lib/Driver/Tools.cpp r1.13 "push back
-pie to the linker if -pie is specified").

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -924,11 +924,6 @@ if(NOT CMAKE_C_COMPILER_ID MATCHES "MSVC" AND NOT OSS_
 	if(CMAKE_VERSION VERSION_GREATER "3.13.999")
 		include(CheckPIESupported)
 		check_pie_supported()
-	else()
-		check_c_linker_flag(-pie LINK_pie_VALID)
-		if(LINK_pie_VALID)
-			set(CMAKE_EXE_LINKER_FLAGS "-pie ${CMAKE_EXE_LINKER_FLAGS}")
-		endif()
 	endif()
 endif()
 
